lms/static/js/commerce/views/receipt_view.js

```diff
@@ -283,6 +283,38 @@ var edx = edx || {};
 })(jQuery, _, _.str, Backbone);
 
 
+// TODO Move this to a library shared with receipt_view.js
```

Comment:
@andy-armstrong how can I consolidate this into a shared library used across these two pages? It's not clear how I would declare such a library so that RequireJS knows how to load it.